e2e-tests/tests/custom-extensions/02-assert.yaml

```diff
@@ -35,15 +35,19 @@ spec:
           value: s3
         - name: STORAGE_ENDPOINT
         - name: STORAGE_REGION
-          value: eu-central-1
+          value: us-east-1
         - name: STORAGE_BUCKET
-          value: pg-extensions
+          value: operator-testing
         - name: INSTALL_EXTENSIONS
         - name: PG_VERSION
         - name: PGDATA_EXTENSIONS
+        - name: STORAGE_DISABLE_SSL
```

Comment:
Is there a way to verify that the parameters were indeed added / passed but not only added to vars?